chart/config/memory.yaml

```diff
@@ -0,0 +1,123 @@
+# Specification of the default memory limits for the kubecf instance
+# groups and jobs.
+#
+# Syntax: See `chart/values.schema.yaml`
+
+# TODO / OPEN ISSUES / NOTES
+#
+# # ## ### ##### Unknown jobs for containers.
+##
+# api:
+#   cloud-controller-ng-ccng-monit-http-healthcheck: 11Mi | ops.run.healthcheck.???
+#   cloud-controller-ng-local-worker-1: 241Mi
+#   cloud-controller-ng-local-worker-2: 250Mi
+#   cloud-controller-ng-nginx: 185Mi
+#
+# singleton-blobstore:
+#   blobstore-nginx: 69Mi	| ops: run.healthcheck.nginx
+#   blobstore-url-signer: 15Mi	| ops: run.healthcheck.url_signer
+#
+# # ## ### ##### One-shot tasks ? Ignore.
+##
+# credhub:
+#   cf-cli-6-linux-credhub-internal-security-group-setup: 4Mi
+#
+# uaa:
+#   cf-cli-6-linux-uaa-internal-security-group-setup: 7Mi
+#
+# # ## ### ##### Dash vs Underscore unknown. Unable to find job.
+##
+# scheduler:
+#       loggr-syslog-binding-cache: 18Mi
+
+# # ## ### ###### ######## #############
+
+limits:
+  memory:
+    api:
+      cc_uploader: 16Mi
```

Comment:
Seems small - did we collect values from the sanbox and increase by 50% as mentioned in #1258 ?

Reply:
In one of the earlier commits I mentioned that these are from my minikube, and without padding.
Getting proper data from the sandbox and pad is in the TODO (See PR description, between the construction warnings).